CMake/telesculptor-external-fletch.cmake

```diff
@@ -44,34 +72,34 @@ ExternalProject_Add(fletch
     -Dfletch_ENABLE_ITK:BOOL=OFF
     -Dfletch_ENABLE_LMDB:BOOL=OFF
     -Dfletch_ENABLE_LevelDB:BOOL=OFF
-    -Dfletch_ENABLE_OpenBLAS:BOOL=OFF    
-    -Dfletch_ENABLE_OpenCV:BOOL=ON
+    -Dfletch_ENABLE_OpenBLAS:BOOL=OFF
+    -Dfletch_ENABLE_OpenCV:BOOL=${TELESCULPTOR_ENABLE_FLETCH_OPENCV}
     -DOpenCV_SELECT_VERSION:STRING=3.4.0
     -Dfletch_ENABLE_OpenCV_FFmpeg:BOOL=ON
     -Dfletch_ENABLE_OpenCV_contrib:BOOL=ON
     -Dfletch_ENABLE_OpenCV_highgui:BOOL=ON
-    -Dfletch_ENABLE_PDAL:BOOL=ON
-    -Dfletch_ENABLE_PNG:BOOL=ON
-    -Dfletch_ENABLE_PROJ:BOOL=ON
+    -Dfletch_ENABLE_PDAL:BOOL=${TELESCULPTOR_ENABLE_FLETCH_PDAL}
+    -Dfletch_ENABLE_PNG:BOOL=${TELESCULPTOR_ENABLE_FLETCH_PNG}
+    -Dfletch_ENABLE_PROJ:BOOL=${TELESCULPTOR_ENABLE_FLETCH_PROJ}
     -Dfletch_ENABLE_PostGIS:BOOL=OFF
     -Dfletch_ENABLE_PostgresSQL:BOOL=OFF
     -Dfletch_ENABLE_Protobuf:BOOL=OFF
-    -Dfletch_ENABLE_Qt:BOOL=ON
+    -Dfletch_ENABLE_Qt:BOOL=${TELESCULPTOR_ENABLE_FLETCH_QT}
     -DBUILD_Qt_MINIMAL:BOOL=ON
     -DQt_SELECT_VERSION:STRING=5.11.2
     -Dfletch_ENABLE_Snappy:BOOL=OFF
     -Dfletch_ENABLE_SuiteSparse:BOOL=ON
```

Comment:
We only need sparesuite to support building Ceres.  I would use `${TELESCULPTOR_ENABLE_FLETCH_CERES}`